zb-classifier.js

```diff
@@ -2201,38 +2201,22 @@ class ZigbeeClassifier {
         return;
       }
 
-      if (node.modelId === 'RC 110' && endpoint == 1) {
+      if (node.modelId === 'RC 110' && endpoint === 1) {
         // endpoint 1 of the RC 110 also controls scenes.
         // The driver treats the scene buttons as level property. Each scene
         // produces a unique level value which is mapped here.
         levelProperty.buttonIndex = 2;
         levelProperty.maximum = 256;
-        this.addEvents(node, {
-          [`4-pressed`]: {
-            '@type': 'PressedEvent',
-            description: `Scene 1`,
-          },
-          [`54-pressed`]: {
-            '@type': 'PressedEvent',
-            description: `Scene 2`,
-          },
-          [`104-pressed`]: {
-            '@type': 'PressedEvent',
-            description: `Scene 3`,
-          },
-          [`154-pressed`]: {
+        const eventCodes = [2, 52, 102, 153, 194, 254];
```

Comment:
I suggest putting the scenes support in a separate function and call from here.

Reply:
done